App/Sagas/NodeOnline.ts

```diff
@@ -0,0 +1,9 @@
+import { put, take } from 'redux-saga/effects'
+import { getType } from 'typesafe-actions'
+import TextileNodeActions from '../Redux/TextileNodeRedux'
+
+export function * onNodeOnline () {
```

Comment:
Wire up all things that should happen once the node is online. This is where refreshing messages trigger happens @sanderpick

Reply:
Ok, cool. To clarify, I think we should call refresh messages when:
- node comes online
- pull down in any of these view: threads, a single thread, notifications

Not sure we should call it when you just view the above views (not pull down). 

In any case, I need to ensure we're not firing off a bunch of these requests in the API layer.

Reply:
K, so with this node online listener, messages are refreshed. We can go through the UI related code and see where else it's being called and make sure it's only in pull to refresh cases.